.github/workflows/CI.yaml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - main
+      - fix/ui-for-feedbackScreen-and-speakingPracticeModule
```

Comment:
Maybe it would be a good idea to make the CI run for pushes on all the branches. I should have thought of that when configuring the CI

Reply:
I did not want to change the CI significantly. I followed the Least Privilege principle only giving permissions which were needed! But yes we should to it for all the branches as it facilitate our work.